lib/src/models/mixins/voteable.dart

```diff
@@ -28,7 +29,10 @@ int _voteStateToIndex(VoteState vote) {
 /// A mixin which provides voting functionality for [Comment] and [Submission].
 mixin VoteableMixin implements RedditBaseInitializedMixin {
   /// The author of the item.
-  String get author => data!['author'];
+  // String get author => data!['author'];
+
+  /// Returns the [Redditor] associated with this item.
+  RedditorRef get author => RedditorRef.name(reddit, data!['author']);
```

Comment:
We can't change the signature of an existing getter. Maybe add a `authorAsRedditor` or `authorRedditor` getter instead?

Reply:
i will just change it to redditor, seems reasonable to me.